app/src/main/java/cm/aptoide/pt/view/FragmentComponent.java

```diff
@@ -103,4 +104,6 @@
   void inject(AppCoinsInfoFragment appCoinsInfoFragment);
 
   void inject(EditorialFragment editorialFragment);
+
+  void inject(DonateDialogFragment donateDialogFragment);
```

Comment:
Should this fragment be injected on both FragmentComponent and ActivityComponent?

Reply:
Fragment component wasn't supposed to be there